front/app/components/admin/Graphs/MultiBarChart/index.tsx

```diff
@@ -99,6 +99,13 @@ const MultiBarChart = <Row,>({
         );
     };
 
+  const tickFormatter = (value: string) => {
+    const label = value.toString(); // Value could be a number, so we want to make sure we are using a string here
+    const limit = 42; // maximum characters before ellipses
+    if (label.length < limit) return value;
+    return `${label.substring(0, limit)}...`;
+  };
```

Comment:
```suggestion
  const tickFormatter = (value: string | number) => {
    const limit = 42; // maximum characters before ellipses
    const label = typeof value === 'number' ? value.toString() : value;

    if (label.length < limit) return label;

    return `${label.substring(0, limit)}...`;
  };
```
If `value` could be a number, the parameter type is wrong. :) (Not sure how it would be a number though, didn't dive in that deep.)

We also have a function called `truncate` for this in `textUtils`.

Reply:
I thought I had seen a case where the label was a number, but I can't seem to find it now :thinking: Seems to be working if I just use "string" so I've removed the additional toString handling :+1: Thanks for reviewing! Didn't realize we had the truncate util either- added that as well :+1: